internal/manager/manager.go

```diff
@@ -92,21 +91,21 @@ func NewManager(dir string, rootConfig *config.RootConfig) *Manager {
 	}
 
 	globalValues, err := values.GetGlobalValues(getRootDirectory(dir))
-	var globalSchema spec.Schema
-	if err == nil && globalValues != nil {
-		globalSchema = *globalValues
+	if err != nil {
+		logger.ErrorF("Failed to get global values: %v", err)
+		return m
 	}
-	globalSchema.Default = make(map[string]any)
-
 	for i := range paths {
 		moduleName := filepath.Base(paths[i])
+		m.errors = m.errors.WithLinterID("manager").WithFilePath(paths[i]).WithModule(moduleName)
```

Comment:
we do not need to rewrite main error list with this fields. it can be passed in liters after
```suggestion
		errorList := m.errors.WithLinterID("manager").WithFilePath(paths[i]).WithModule(moduleName)
```

Reply:
maybe we need a getter or smthing like to prevent this behaviour later

Reply:
fixed